### PR TITLE
Add normative references for all properties

### LINKED ISSUE #13: Normative Reference concerns
Unicode Consortium documents that are essential to the interpretation of this specification should be normatively referenced. If they are not already listed in ECMA-262 [clause 3](https://tc39.github.io/ecma262/#sec-normative-references) they need to be added (also see https://github.com/tc39/ecma262/issues/726). Those additions should be listed as part of the proposed spec. text.

The documents that appear to be in need of normative references are: UTS #18 and UAX #44

A consistent style for referencing Unicode Consortium documents should be used throughout ECMA-262. Existing references in ECMA-262 typically spell out the full document title, such as: Unicode Standard Annex #31, Identifier and Pattern Syntax. This proposal uses abbreviated references such as: UTS18.

### PR DESCRIPTION
**Pull request by mathiasbynens**
Closes #13.
Closes #24.

### CODE REVIEW

**Comment by littledan** [comment]
Awesome, thanks for this!

File: spec.html [hunk 1]
```diff
@@ -126,81 +126,81 @@
...
+    <li><a href="http://unicode.org/reports/tr44/#CWKCF">`Changes_When_NFKC_Casefolded`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWT">`Changes_When_Titlecased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#CWU">`Changes_When_Uppercased`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Composition_Exclusion">`Composition_Exclusion`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Dash">`Dash`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Default_Ignorable_Code_Point">`Default_Ignorable_Code_Point`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Deprecated">`Deprecated`</a></li>
+    <li><a href="http://unicode.org/reports/tr44/#Diacritic">`Diacritic`</a></li>
+    <li><a href="http://unicode.org/reports/tr51/#Emoji_Properties">`Emoji`</a></li>
+    <li><!-- Note: by the time this makes it into ECMA-262, the draft defining `Emoji_Component` will likely have been published already. Update the following link accordingly when that happens. --><a href="http://unicode.org/reports/tr51/proposed.html#Emoji_Properties">`Emoji_Component`</a></li>
```
Comment (littledan) [inline spec.html:156]: At the March 2017 TC39 meeting, @bterlson mentioned he had some ideas for the procedure for evolution of adding more properties. We could use this as a test case, leaving it out and adding it back in in round two. What do you think? How certain is this proposal to be accepted?

Comment (mathiasbynens) [reply spec.html:156]: > How certain is this proposal to be accepted?

All I know is that `Emoji_Component` has been included in the past two draft snapshots.

> We could use this as a test case, leaving it out and adding it back in in round two. 

I’d be okay with that. @bterlson, what do you think?

Comment (bterlson) [reply spec.html:156]: Seems fine, but I'm not actually sure what @littledan is referring to. Can someone remind me what I said?

Comment (littledan) [reply spec.html:156]: @wycats raised the issue that we'll want to add more features in the future here, so maybe we shouldn't include the property list in the spec. I said I'd prefer that the spec be specific, to coordinate between implementations. IIRC you said that you had some ideas for how we could handle this in the future, but didn't get a chance to say.

**Pull request merged**